Comp2023/src/main/java/frc/robot/commands/DriveBalance.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.wpilibj.RobotState;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Swerve;
+
+/**
+ *
+ */
+public class DriveBalance extends CommandBase
+{
+  private Swerve m_swerve;
+  private Timer  m_timer = new Timer( );
+
+  public DriveBalance(Swerve swerve)
+  {
+    m_swerve = swerve;
+
+    addRequirements(m_swerve);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize( )
+  {
+    DataLogManager.log("Running Drive Balance!");
+    m_timer.reset( );
+    m_timer.start( );
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute( )
+  {
+    m_swerve.driveBalance( );
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted)
+  {
+    m_timer.stop( );
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished( )
+  {
+    if (m_timer.hasElapsed(15) || RobotState.isDisabled( ))

```

Comment:
This can be much simpler with just
   return (m_timer.hasElapsed(15) || RobotState.isDisabled());

Since that will evaluate to true when either is true, and false otherwise. All on one line.